java/src/main/java/sh/platform/languages/sample/ElasticsearchSample.java

```diff
@@ -0,0 +1,87 @@
+package sh.platform.languages.sample;
+
+import org.elasticsearch.action.admin.indices.refresh.RefreshRequest;
+import org.elasticsearch.action.admin.indices.refresh.RefreshResponse;
+import org.elasticsearch.action.delete.DeleteRequest;
+import org.elasticsearch.action.index.IndexRequest;
+import org.elasticsearch.action.search.SearchRequest;
+import org.elasticsearch.action.search.SearchResponse;
+import org.elasticsearch.client.RequestOptions;
+import org.elasticsearch.client.RestHighLevelClient;
+import org.elasticsearch.index.query.QueryBuilders;
+import org.elasticsearch.search.SearchHit;
+import org.elasticsearch.search.builder.SearchSourceBuilder;
+import sh.platform.config.Config;
+import sh.platform.config.Elasticsearch;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.function.Supplier;
+
+import static java.util.concurrent.ThreadLocalRandom.current;
+
+public class ElasticsearchSample implements Supplier<String> {
+
+    @Override
+    public String get() {
+        StringBuilder logger = new StringBuilder();
+
+        // Create a new config object to ease reading the Platform.sh environment variables.
+        // You can alternatively use getenv() yourself.
+        Config config = new Config();
+
+        Elasticsearch elasticsearch = config.getCredential("elasticsearch", Elasticsearch::new);
+
+        // Create an Elasticsearch client object.
+        RestHighLevelClient client = elasticsearch.get();
+
+        try {
+
+            String index = "animals";
+            String type = "mammals";
+            // Index a few document.
+            final List<String> animals = Arrays.asList("dog", "cat", "monkey", "horse");
+            for (String animal : animals) {
+                Map<String, Object> jsonMap = new HashMap<>();
+                jsonMap.put("name", animal);
+                jsonMap.put("age", current().nextInt(1, 10));
+                jsonMap.put("is cute?", current().nextBoolean());
```

Comment:
Is a ? legal here?  Seems like that's asking for trouble using a ? in a field name...

Reply:
It works, but I'll remove it, you're right it does not look cool to put `?` in the field.